tests/specs/alone/iFrameApiRecording.spec.ts

```diff
@@ -1,4 +1,11 @@
 import { ensureOneParticipant } from '../../helpers/participants';
+import { setTestProperties } from "../../helpers/TestProperties";
+
+setTestProperties(__filename, {
+    useIFrameApi: true,
+    // TODO: does this actually use webhook proxy?
```

Comment:
Well it does even call some webhook proxy methods. So I think it definitely does.

Reply:
Thanks, I blindly added these comments as I split this into two separate flags. I've cleaned them all up per your review!